### ISSUE TITLE
val_loss is larger than loss on two datasets  

### ISSUE THREAD

**Issue by f-rumblefish** [qqwweee/keras-yolo3#347]
I am beginning to train two datasets via yolo3. But, both of them have low `loss` and very high `val_loss` at the first three epochs. Further, the decreasing rate on `val_loss` is slow. Is this normal in training our own datasets? What are the proper ways to do further investigation?

Case 1. Train - 1618 images, Validation - 179 images, Number of Classes - 6

```
Train on 1618 samples, val on 179 samples, with batch size 8.
Epoch 1/10
202/202 [==============================]   - 15762s 78s/step - loss: 705.8173 - val_loss: 4766.2180
Epoch 2/10
202/202 [==============================]   - 25779s 128s/step - loss: 94.2822 - val_loss: 4482.7397
Epoch 3/10
202/202 [==============================]   - 10060s 50s/step - loss: 63.8548 - val_loss: 4340.8203
```

Case 2: Train - 2404 images, Validation - 267 images, Number of Classes - 13

```
Train on 2404 samples, val on 267 samples, with batch size 8.
Epoch 1/10
300/300 [==============================] - 11371s 38s/step - loss: 601.9105 - val_loss: 4815.2585
Epoch 2/10
300/300 [==============================] - 11322s 38s/step - loss: 87.9751 - val_loss: 4587.5893
Epoch 3/10
300/300 [==============================] - 12102s 40s/step - loss: 65.5565 - val_loss: 4553.0205
```

I have tried to set `print_loss` as `True` in `yolo_loss` under `model.py`. It seems to me that `confidence_loss` is pretty large in both cases.

**Comment by sxyxf66** [comment]
@f-rumblefish My loss is also so big. Do you know why? Thanks

**Comment by f-rumblefish** [comment]
Try create_model with load_pretrained=False and see if it works or not.
